packages/connectivity/src/scp-cf/destination/destination-service.ts

```diff
@@ -164,6 +173,46 @@ export async function fetchDestination(
     options
   );
 }
+/**
+ * Fetches a certificate from the subaccount given a name.
+ * @param destinationServiceUri - The URI of the destination service
+ * @param token - The access token for destination service.
+ * @param certificateName - Name of the Certificate to be fetched
+ * @returns A Promise resolving to the destination
+ * @internal
+ */
+export async function fetchSubaccountCertificate(
```

Comment:
Question: do we only support sub account certificates, or is there an instance certificate that is used by instance destination?

Reply:
As a first version I wanted to included only account certificate because I was wondering if people upload certificates on the destination service instance level - this feels odd to me, because I would expect that a truststore is related to an account and not some service subscription.

Reply:
If it is technically possible why not support both even though the use cases might be unknown?

Reply:
The question would be, what if both resolve? Which certificate wins, then we would need a selection strategy. I wanted to keep it simple in the first version.

Reply:
I see your point. I see that you now implemented subscriberfirst - should we make this configurable? (on a separate PR then?)

Reply:
I implemented now subaccount first, which is I guess the usecase in 99% of the cases. Only if not found there it looks for the instance - I would wait if somebody is unhappy with the situation and opens and issue.